packages/common/src/store/buy-usdc/sagas.ts

```diff
@@ -117,6 +127,50 @@ function* purchaseStep({
   return { newBalance }
 }
 
+function* transferStep({
+  wallet,
+  userBank,
+  amount
+}: {
+  wallet: Keypair
+  userBank: PublicKey
+  amount: BN
+}) {
+  const audiusBackendInstance = yield* getContext('audiusBackendInstance')
+  const feePayer = yield* select(getFeePayer)
+  if (!feePayer) {
+    throw new Error('Missing feePayer unexpectedly')
+  }
+  const feePayerOverride = new PublicKey(feePayer)
+  const recentBlockhash = yield* call(getRecentBlockhash, audiusBackendInstance)
+
+  const transferTransaction = yield* call(
+    createTransferToUserBankTransaction,
+    audiusBackendInstance,
+    {
+      wallet,
+      userBank,
+      mint: 'usdc',
+      amount,
+      memo: 'In-App $USDC Purchase: Link by Stripe',
+      feePayer: feePayerOverride,
+      recentBlockhash
+    }
+  )
+  transferTransaction.partialSign(wallet)
+  console.debug(`Starting transfer transaction...`)
+  const { res, error } = yield* call(relayTransaction, audiusBackendInstance, {
+    transaction: transferTransaction
+  })
+  if (error) {
+    console.debug(
+      `Transfer transaction stringified: ${JSON.stringify(transferTransaction)}`
+    )
+    throw new Error(`Transfer transaction failed: ${error}`)
+  }
+  console.debug(`Transfer transaction succeeded: ${res}`)
```

Comment:
Just confirming you meant to leave the debug statements in?

Reply:
Yep!